sample/src/main/java/com/maxkeppeler/sample/MainActivity.kt

```diff
@@ -656,7 +666,19 @@ class MainActivity : AppCompatActivity() {
 
         CustomSheet().show(this) {
             title("Custom Example")
-            onPositive("Cool")
+            style(SheetStyle.values().random())
+            onPositive(
+                    "Cool",
+                    R.drawable.ic_apple) { }
+            onNegative(
+                    "Cancel",
+                    R.drawable.ic_clear) { }
+            withCoverImage(Image("https://cdn.wallpapersafari.com/11/17/LjhbqX.jpg") {
+                // For placeholder, error, fallback drawable and other image loading configs
+                crossfade(300)
+            })
+            content("It is set in a world where humanity lives inside cities surrounded by enormous walls that protect them from gigantic man-eating humanoids referred to as Titans; the story follows Eren Yeager, who vows to exterminate the Titans after a Titan brings about the destruction of his hometown and the death of his mother.")
+            withIconButton(IconButton(R.drawable.ic_send)) { /* Will not automatically dismiss the sheet. */ }
```

Comment:
I believe those common use cases were already shown enough in all the other examples. Since the focus is on a custom layout, there's no need to showcase all the other options again. At a later point I will just make the custom layout more complex to showcase, that anything someone wish can be done by themselves.

Reply:
Cool